manifests/init.pp

```diff
@@ -130,6 +131,7 @@
   Optional[Hash]                 $high_availability               = undef,
   Optional[Hash]                 $logging                         = undef,
   Optional[Hash]                 $letsencrypt                     = undef,
+  Optional[Hash]                 $package                         = undef,
```

Comment:
can you enforce the datatype a bit? What about this:
```suggestion
  Hash[String[1],String[1]]      $package                         = {},
```

if possible, the default for arrays and hashes should always be an empty hash/array

Reply:
I've kept it optional with undef as the default as this is truly an optional param (as are the other hashes). This will also ensure that it won't (at least the header/description) be rendered into the template when it's not supplied.

I've adjusted the data type to `Optional[Hash[String[1], Scalar]]`.